assets/src/scss/layout/_block-classes.scss

```diff
@@ -0,0 +1,45 @@
+.wp-block-query-pagination {
```

Comment:
For now it's the only style, so I added it directly. In the future we could turn this into a block style, once we use the block in the editor. Or when we add another back end generated pagination style.